ESLintの警告を消す

```bash
$ eslint src/ --ext jsx

/home/huequica/Github.com/huequica/Gatsbied_portfolio/src/pages/about.jsx
  10:5  warning  Imported JSX component layout must be in PascalCase or SCREAMING_SNAKE_CASE    react/jsx-pascal-case
  11:7  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case
  14:5  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case

/home/huequica/Github.com/huequica/Gatsbied_portfolio/src/pages/aboutItems/myname.jsx
  10:7  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case

/home/huequica/Github.com/huequica/Gatsbied_portfolio/src/pages/dev.jsx
  53:5  warning  Imported JSX component layout must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case

/home/huequica/Github.com/huequica/Gatsbied_portfolio/src/pages/index.jsx
   7:5  warning  Imported JSX component layout must be in PascalCase or SCREAMING_SNAKE_CASE    react/jsx-pascal-case
  10:7  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case

/home/huequica/Github.com/huequica/Gatsbied_portfolio/src/pages/works.jsx
   7:5  warning  Imported JSX component layout must be in PascalCase or SCREAMING_SNAKE_CASE    react/jsx-pascal-case
   8:7  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case
  11:5  warning  Imported JSX component elements must be in PascalCase or SCREAMING_SNAKE_CASE  react/jsx-pascal-case

✖ 10 problems (0 errors, 10 warnings)
```

これをなんとかする

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "build": "gatsby build",
     "develop": "gatsby develop",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "lint": "eslint src/",
+    "lint": "eslint src/ --ext jsx",
     "start": "npm run develop",
     "serve": "gatsby serve",
     "clean": "gatsby clean",

--- a/src/bulmaElements/index.js
+++ b/src/bulmaElements/index.js
@@ -1,5 +1,5 @@
-import * as elements from './elements/elements';
-import * as layout from './layout/layout';
+import * as Elements from './elements/elements';
+import * as Layout from './layout/layout';
 import Column from './column';
 
-export { elements, layout, Column };
+export { Elements, Layout, Column };

--- a/src/pages/about.jsx
+++ b/src/pages/about.jsx
@@ -7,11 +7,11 @@ import MyName from './aboutItems/myname';
 
 const About = () => (
   <Layout>
-    <Bulma.layout.Hero colorStyle='is-primary'>
-      <Bulma.elements.Title>/about</Bulma.elements.Title>
-    </Bulma.layout.Hero>
+    <Bulma.Layout.Hero colorStyle='is-primary'>
+      <Bulma.Elements.Title>/about</Bulma.Elements.Title>
+    </Bulma.Layout.Hero>
 
-    <Bulma.elements.Title subTitle>{ 'huequica.xyzについて' }</Bulma.elements.Title>
+    <Bulma.Elements.Title subTitle>{ 'huequica.xyzについて' }</Bulma.Elements.Title>
 
     Haru Yakumoが個人で運営しているポートフォリオみたいなサイトです。<br />
     GatsbyJSによって記述されています。

--- a/src/pages/aboutItems/myname.jsx
+++ b/src/pages/aboutItems/myname.jsx
@@ -7,7 +7,7 @@ import NamesDescription from './namesDescription';
 const MyName = () => {
   return (
     <>
-      <Bulma.elements.Title subTitle>Haru Yakumoについて</Bulma.elements.Title>
+      <Bulma.Elements.Title subTitle>Haru Yakumoについて</Bulma.Elements.Title>
 
       <Bulma.Column Parent>
         <Bulma.Column size='is-4'>

--- a/src/pages/aboutItems/namesDescription.jsx
+++ b/src/pages/aboutItems/namesDescription.jsx
@@ -3,7 +3,7 @@ import * as Bulma from '@bulma';
 import AniLink from 'gatsby-plugin-transition-link/AniLink';
 
 // TitleModule alias
-const Title = Bulma.elements.Title;
+const Title = Bulma.Elements.Title;
 
 const NamesDescription = () => {
   return (

--- a/src/pages/dev.jsx
+++ b/src/pages/dev.jsx
@@ -3,7 +3,7 @@ import Layout from '@layout';
 import * as Bulma from '@bulma';
 
 // TitleModule alias
-const Title = Bulma.elements.Title;
+const Title = Bulma.Elements.Title;
 
 const FrontEnd = () => (
   <>
@@ -50,9 +50,9 @@ const Descriptions = () => (
 
 const Dev = () => (
   <Layout>
-    <Bulma.layout.Hero colorStyle='is-primary'>
+    <Bulma.Layout.Hero colorStyle='is-primary'>
       <Title>/dev</Title>
-    </Bulma.layout.Hero>
+    </Bulma.Layout.Hero>
 
     <Descriptions />
   </Layout>

--- a/src/pages/index.jsx
+++ b/src/pages/index.jsx
@@ -4,11 +4,11 @@ import * as Bulma from '@bulma';
 
 const IndexPage = () => (
   <Layout disableMargin={ true }>
-    <Bulma.layout.Hero colorStyle='is-info' size='is-fullheight'
+    <Bulma.Layout.Hero colorStyle='is-info' size='is-fullheight'
       subtleGradient={ true }
     >
-      <Bulma.elements.Title>Haru Yakumo's Portfolio Site</Bulma.elements.Title>
-    </Bulma.layout.Hero>
+      <Bulma.Elements.Title>Haru Yakumo's Portfolio Site</Bulma.Elements.Title>
+    </Bulma.Layout.Hero>
   </Layout>
 );
 

--- a/src/pages/works.jsx
+++ b/src/pages/works.jsx
@@ -13,11 +13,11 @@ const Box = ({children}) => <div className='box'>{ children }</div>;
 
 const Works = () => (
   <Layout>
-    <Bulma.layout.Hero colorStyle='is-primary'>
-      <Bulma.elements.Title>/worsks</Bulma.elements.Title>
-    </Bulma.layout.Hero>
+    <Bulma.Layout.Hero colorStyle='is-primary'>
+      <Bulma.Elements.Title>/worsks</Bulma.Elements.Title>
+    </Bulma.Layout.Hero>
     
-    <Bulma.elements.Title># 個人の制作物</Bulma.elements.Title>
+    <Bulma.Elements.Title># 個人の制作物</Bulma.Elements.Title>
 
     <Box> { /* @HikakinSymmetry */ }
       <Bulma.Column Parent>
@@ -26,9 +26,9 @@ const Works = () => (
         </Bulma.Column>
 
         <Bulma.Column>
-          <Bulma.elements.Title subTitle>
+          <Bulma.Elements.Title subTitle>
             # ヒカキンシンメトリーbot(新)
-          </Bulma.elements.Title>
+          </Bulma.Elements.Title>
 
           <p className='mb-3'>
             <Intent link='https://twitter.com/hikakin_sym'>ヒカキンシンメトリーbot</Intent>の稼働が止まっていたので再実装して公開したbot<br />
@@ -37,11 +37,11 @@ const Works = () => (
 
           <div className='buttons are-medium'>
             <Intent link='https://github.com/HIKAKINSymmetry/nodeBot'>
-              <Bulma.elements.Button isRounded color='is-dark'>GitHub</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-dark'>GitHub</Bulma.Elements.Button>
             </Intent>
 
             <Intent link='https://twitter.com/hikakinsymmetry'>
-              <Bulma.elements.Button isRounded color='is-info'>Twitter</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-info'>Twitter</Bulma.Elements.Button>
             </Intent>
           </div>
 
@@ -57,9 +57,9 @@ const Works = () => (
         </Bulma.Column>
 
         <Bulma.Column>
-          <Bulma.elements.Title subTitle>
+          <Bulma.Elements.Title subTitle>
             soundrop_OtogibaraEra
-          </Bulma.elements.Title>
+          </Bulma.Elements.Title>
 
           <p className='mb-3'>
             にじさんじに所属(していた)御伽原江良の声を出すだけのおもちゃ<br />
@@ -69,11 +69,11 @@ const Works = () => (
 
           <div className='buttons are-medium'>
             <Intent link='https://github.com/huequica/soundrop_OtogibaraEra'>
-              <Bulma.elements.Button isRounded color='is-dark'>GitHub</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-dark'>GitHub</Bulma.Elements.Button>
             </Intent>
 
             <Intent link='https://suspicious-curran-504da5.netlify.app/'>
-              <Bulma.elements.Button isRounded color='is-success'>Sample</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-success'>Sample</Bulma.Elements.Button>
             </Intent>
           </div>
 
@@ -82,7 +82,7 @@ const Works = () => (
       </Bulma.Column>
     </Box>
 
-    <Bulma.elements.Title># OSS Contribution</Bulma.elements.Title>
+    <Bulma.Elements.Title># OSS Contribution</Bulma.Elements.Title>
 
     <Box> { /* @Types/twit */ }
       <Bulma.Column Parent>
@@ -90,9 +90,9 @@ const Works = () => (
         <Bulma.Column size='is-3' /> { /* Empty column */ }
 
         <Bulma.Column>
-          <Bulma.elements.Title subTitle>
+          <Bulma.Elements.Title subTitle>
             @types/twit
-          </Bulma.elements.Title>
+          </Bulma.Elements.Title>
 
           <p className='mb-3'>
             <Intent link='https://www.npmjs.com/package/twit'>twit</Intent>の型定義が古くて1枚の画像の取得しかできなそうだったので追加したPR
@@ -100,11 +100,11 @@ const Works = () => (
 
           <div className='buttons are-medium'>
             <Intent link='https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53197'>
-              <Bulma.elements.Button isRounded color='is-dark'>GitHub PR</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-dark'>GitHub PR</Bulma.Elements.Button>
             </Intent>
 
             <Intent link='https://www.npmjs.com/package/@types/twit'>
-              <Bulma.elements.Button isRounded color='is-success'>npmjs.com</Bulma.elements.Button>
+              <Bulma.Elements.Button isRounded color='is-success'>npmjs.com</Bulma.Elements.Button>
             </Intent>
           </div>
 

